NAS2D/Mixer/Mixer.h

```diff
@@ -19,18 +19,16 @@ namespace NAS2D
 	class Sound;
 	class Music;
 
-
 	/**
-	 * Mixer base class.
-	 *
-	 * Provides a standard Mixer interface. The base Mixer can be used
-	 * but will act as a NULL interface.
-	 */
+	 * @brief Mixer base class.
```

Comment:
We're using [`JAVADOC_AUTOBRIEF = YES`](https://www.doxygen.nl/manual/config.html#cfg_javadoc_autobrief) in the `Doxyfile`, so it's possible to omit explicit `@brief`. I suppose this becomes a question of style now. Use of `\brief` was removed in PR #856.

----

We should probably settle on a convention between `@` and `\` for Doxygen tags. I think I prefer the `@`, though the code base has typically used `\`.

Reply:
The use of the `@` is a consequence of the `.editorconfig` file. Visual Studio auto-generates the documentation via `Ctrl+/` and inserts the `@`.

The happy accident this provides is that Intellisense correctly displays the comments...sort of. See below.

Reply:
@ldicker83 : You should probably take a look at this comment.

----

I noticed newer Doxygen examples tend to use `@` rather than `\` for Doxygen commands. Should we perhaps consider switching style?

----

Should we include `@brief` explicitly, or rely on `JAVADOC_AUTOBRIEF = YES`? I would prefer to omit explicit commands if they don't need to be specified, and instead rely on the auto brief feature.

Reply:
There hasn't been any comment from Leeor. Unless he wants to chime in and say otherwise, I would say remove `\brief` and rely on auto brief. That would at least be consistent with current usage. No reason to reject something for a part that wasn't changed. :wink:

Besides, I think I like the shorter format of auto brief. The formatting of the documentation pretty much dictates what part is the brief anyway.

Reply:
Just to clarify, Remove all `\brief` tags in favor of just a description?

Reply:
> [`JAVADOC_AUTOBRIEF`](https://www.doxygen.nl/manual/config.html#cfg_javadoc_autobrief)
> 
> If the JAVADOC_AUTOBRIEF tag is set to YES then doxygen will interpret the first line (until the first dot) of a Javadoc-style comment as the brief description. If set to NO, the Javadoc-style will behave just like regular Qt-style comments (thus requiring an explicit @brief command for a brief description.)
> 
> The default value is: NO.

We have:
`JAVADOC_AUTOBRIEF = YES`

So yes, remove the `\brief`. The description on it's own will already be split into a brief and the rest.

Reply:
Ok. I am in the middle of doing that and as usual, Intellisense has to be a butt. Removing the brief tag and just leaving the javadoc-style comment like so (extreme example):

```cpp
/**
* Mixer base class.
* Mixer base class.
* Mixer base class.
* Mixer base class.
* Mixer base class.
*/
```
Causes the intermediate optional `*`'s to show up:

```cpp
* Mixer base class
* Mixer base class
* Mixer base class
* Mixer base class
* Mixer base class
```

Removing the intermediate `*`'s fixes the problem.

Reply:
Probably nothing, but I noticed the `*` are not all vertically aligned. Does that make any difference? It's possible it may be very sensitive to formatting.

Reply:
Doesn't make a difference.

Reply:
The interpreter (and specification) only looks for the pattern:
```cpp
/**
... text...
*/
```
It doesn't care what's in between.